app/views/notifiers/course/consolidated_opening_reminder_notifier/opening_reminder/course_notifications/email.html.slim

```diff
@@ -0,0 +1,13 @@
+- course = @object
+- items_hash = Course::LessonPlan::Item.upcoming_items_from_course_by_type(course)
+/ Lesson plan item start at times could have been changed between the time the mailer job was
+/ enqueued and the time this view partial is called.
+/ Raise an exception if there are no longer any items so the consolidated email doesn't get sent.
+- raise 'No opening lesson plan items' if items_hash.empty?
```

Comment:
Would suggest not to raise the error here, but rather in the `ConsolidatedOpeningReminderMailer`, because this file should deal with the display, rather than the logic of halting the mailer. 

With the custom mailer, we can construct the items_hash there, terminate if empty, and send the email if `items_hash` has objects.